### DISCUSSION TITLE
[Spoiler] The floating islands

### DISCUSSION THREAD

**Discussion by CloakedGhost** [Monika-After-Story/MonikaModDev#8896] [appreciation]
This is not an issue, just a big "good job" and thank you to the devs. I love the new floating islands and I can't wait to see what Monika will add in the future!

**Comment by lunulae** [comment]
There are quite a few exciting surprises coming~ Glad you're looking forward to them!

**Comment by ThePotatoGuy** [comment]
[closing as appreciated]
